src/containers/ViewerPanel/index.tsx

```diff
@@ -261,13 +264,25 @@ class ViewerPanel extends React.Component<ViewerPanelProps, ViewerPanelState> {
         setIsLooping(!isLooping);
     }
 
+    public handleIncomingConvertedFile(data: TrajectoryFileInfo) {
+        const { receiveConvertedFile, conversionProcessingData } = this.props;
+        const fileId = conversionProcessingData.fileId + ".simularium";
+        const hasTitle =
+            data.trajectoryTitle !== undefined &&
+            data.trajectoryTitle.length > 0;
+        const title = hasTitle ? data.trajectoryTitle : fileId;
+        receiveConvertedFile({
+            name: fileId,
+            title: title,
+        });
```

Comment:
This incoming file _should_ have a title, but it will definitely have a fileId, so we can default to that. Having a title is how we have been checking that a file is networked and not local.

Reply:
what are these two things used for? and how are they different? just curious why they're set to different things (although they may be both fieldId in some cases it seems

Reply:
The existing check for whether a file is networked and not a local file is that it has a "title" stored in redux.

The trajectory that comes back from the server after conversion has both a fileName and title, the fileName is typically the uuid provided during conversion and the title is (currently) derived from the file name that the user uploaded. In future we will let them choose their preferred title.

I put the fileId as a backup/failsafe for the title to ensure that the trajectory is recognized as a networked file and not a local file if for some reason it comes back from the server with that field empty, but it shouldn't!